setup.py

```diff
@@ -31,15 +31,15 @@
 
 kwargs = {
     'name': 'rocker',
-    'version': '0.2.3',
+    'version': '0.2.4',
```

Comment:
Same comments here about changing the version and the unrelated changes.

Reply:
Sorry, I don't get it, the other version change was for the minor number but here I changed the patch version. Do you mean that I should let the version to 0.2.3?